src/context/favorites/Actions.ts

```diff
@@ -0,0 +1,6 @@
+import { ActionTypes } from 'enums/ActionTypes';
+import { Action } from './models';
+
+const ADD = (payload: string): Action => ({ type: ActionTypes.ADD, payload });
```

Comment:
```suggestion
const addAction = (payload: string): Action => ({ type: ActionTypes.ADD, payload });
```
cc: @dkaczmarek-cc @kamil-borecki-cc, please let me know if i'm wrong, but I would like to suggest that method should be defined by uppercase letters here.

Reply:
We changed that :+1: